### ISSUE TITLE
Create a way to count of the PR merged to private repo's

### ISSUE THREAD

**Issue by LundPiyush** [anuraghazra/github-readme-stats#2156]
### Is your feature request related to a problem? Please describe.

Current params are good but need to add one more param that should add the number of PR's merged to private repository to the total PR 

### Describe the solution you'd like

count_pr_private_repo=true
Default value will be set as false . But one can change as true.

**Comment by rickstaa** [comment]
@LundPiyush, thanks for your question. We don't have access to your private data. As a result, you need to [deploy your own Vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance) for the private stats (other than the commits) to show up (see https://github.com/anuraghazra/github-readme-stats/discussions/1449 for more information).

**Comment by LundPiyush** [comment]
@rickstaa thanks . So total PR merged I need to deploy my own vercel instance as count_private won't count my PR merged on private repository ?

**Comment by rickstaa** [comment]
@LundPiyush That's correct. Please let us know if we could update the documentation to clarify unclarities (see https://github.com/anuraghazra/github-readme-stats#adding-private-contributions-count-to-total-commits-count).

**Comment by JacobKerames** [comment]
Hello @rickstaa,

I apologize if this has been answered elsewhere but I've looked in several discussions and issues and have been unable to find a straightforward answer.

I've deployed my own vercel instance for the github readme stats card. I've also included the repo scope in my PAT. However, still only private commits are included in the stats. I am particularly trying to include pull requests and issues in private repos in the stats count. Am I missing a step?

Thanks,

Jake
